tasks/elasticsearch-parameters.yml

```diff
@@ -13,12 +13,12 @@
   when: es_config['http.port'] is not defined
 
 - name: debug message
-  debug: msg="WARNING - It is recommended you specify the parameter 'transport.tcp.port'"
-  when: es_config['transport.tcp.port'] is not defined
+  debug: msg="WARNING - It is recommended you specify the parameter 'transport.port'"
+  when: es_config['transport.port'] is not defined
```

Comment:
This will be a bit confusing for someone using 6.x since it will say to set an option that is not available. 

I think it would be better to just remove this debug line altogether since it is already mentioned in the readme.